### ISSUE TITLE
Missing recursive $touch method in 2.0

### ISSUE THREAD

**Issue by blowsie** [vuelidate/vuelidate#608]
In the current next branch, there is no method available to call touch on the entire form state, as you can with the current version.

```
<script>
import { ref, reactive } from '@vue/composition-api'
import { email, required } from '@vuelidate/validators'
import useVuelidate from '@vuelidate/core'
import useLogin from '@/composables/auth/useLogin'
export default {
  name: 'LoginForm',
  setup() {
    // Wrapping this in reactive since the plugin has a bug with .value
    const loginService = reactive(useLogin())
    const username = ref()
    const password = ref()
    const submit = () => {
      validation.$touch();
      loginService.login(username.value, password.value)
    }

    let validation = useVuelidate(
      {
        username: { required, email },
        password: { required }
      },
      { username, password }
    )

    return {
      username,
      password,
      submit,
      loginService,
      validation
    }
  }
}
</script>
```

When calling submit, `validation` is defined with the vuelidate state, but `$touch` is only available on each of the nested items, in this case `username` and `password`.

--- 
There is no equivalent of this https://github.com/vuelidate/vuelidate/blob/master/src/index.js#L131 currently.

**Comment by shentao** [comment]
Hey! Thanks for the report! I'm aware of the issue. Planning to do some work on it this weekend.

**Comment by dobromir-hristov** [comment]
Fixed in #626
